### PR TITLE
Update committe history

### PR DESCRIPTION
**Pull request by lindenwells**
Changes from the SGM held on the 19th of May 2022 and from Tom Cranitch's resignation in January 2022.

### CODE REVIEW

**Comment by LimaoC** [comment]
Hmm yeah, not sure about changing "Puisne Committee" -> "General Committee" just yet - the constitution hasn't technically been updated yet to reflect these changes, the PR is still yet to be merged (#35)

File: committee_history.md [hunk 1]
```diff
@@ -125,10 +125,13 @@
 - President: Tom Malcolm
 - Secretary: Limao Chang
 - Treasurer: Andrew Brown
-- Puisne Committee:
+- General Committee:
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply committee_history.md:128]: Most of these committee members were elected as Puisne, and the resigning members were never general committee members, so it's inaccurate to update this header.

**Comment by andrewj-brown** [comment]
alright it's now Puisne Committee (but renamed from sgm), are we satisfied that this accurately represents the title(s) held by committee members?

**Review by nicklambourne — APPROVED** [review]
LGTM

**Pull request merged**